chapters/ch01.md

```diff
@@ -36,7 +36,7 @@ Nhóm bạn có thể đưa ra rất nhiều ý tưởng như:
 
 > * Train the algorithm longer, by running more gradient descent iterations.
 
-* Huấn luyện thuật toán lâu hơn bằng cách chạy nhiều vòng lặp gradient descent hơn. 
+* Huấn luyện thuật toán lâu hơn bằng cách chạy thêm nhiều vòng lặp hạ dốc. 
```

Comment:
Mình thêm một note ở đây về việc cần dịch lại cụm "gradient descent". Từ "gradient" được dịch thành "gradien" trong nhiều tài liệu tiếng Việt, nó có ý nghĩa toán học rõ ràng, không thể chuyển sang "dốc" được.

https://vi.wikipedia.org/wiki/Gradien

Reply:
Mình đã mở thảo luận ở #389